api/webhooks/models.py

```diff
@@ -1,10 +1,24 @@
-from core.models import AbstractBaseExportableModel
+from core.models import AbstractBaseExportableModel, SoftDeleteExportableModel
 from django.db import models
 
 
-class AbstractBaseWebhookModel(AbstractBaseExportableModel):
+class AbstractBaseWebhookModel(models.Model):
     url = models.URLField()
     secret = models.CharField(max_length=255, blank=True)
 
     class Meta:
         abstract = True
+
+
+class AbstractBaseExportableWebhookModel(
+    AbstractBaseWebhookModel, AbstractBaseExportableModel
+):
+    class Meta:
+        abstract = True
+
+
+class AbstractBaseSoftDeleteExportableWebhookModel(
```

Comment:
Where is this being used?

Reply:
Here: https://github.com/Flagsmith/flagsmith/pull/1975/files#diff-c2f900907b02278b20496d66a689e7b7b21abb20fc46c850417652e7a8eddaf8L13